packages/app/src/app/pages/Dashboard/Components/Folder/FolderCard.tsx

```diff
@@ -67,8 +70,12 @@ export const FolderCard: React.FC<FolderItemComponentProps> = ({
     >
       {!isNewFolder ? (
         <IconButton
+          css={{
+            borderRadius: '2px',
+            cursor: 'default',
+          }}
```

Comment:
Is this one of the cases where if we override the `IconButton` styles it also affects the editor? We could create an abstraction of this component, like `IconButtonDash` or rename the default to `Round` or something. Or create a variant? Just emptying my thoughts here.

Reply:
Added a `variant` prop for the IconButton